src/cook-web/src/components/outline-tree/index.tsx

```
@@ -1,243 +1,351 @@
+                actions.setFocusId(props.item.id || '')
+              }}
+            />
+          </span>
+          {(props.item?.depth || 0 > 0) && (
+            <div className='flex items-center space-x-1'>
+              {cataData[props.item.id!]?.status == 'saving' && (
+                <Loading className='h-4 w-4' />
+              )}
+            </div>
+          )}
+          {(props.item?.depth || 0 > 0) && (
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)